Project.toml

```diff
@@ -4,6 +4,7 @@ version = "0.8.3"
 
 [deps]
 Compat = "34da2185-b29b-5c13-b0c7-acf172513d20"
+Documenter = "e30172f5-a6a5-5a46-863b-614d45cd2de4"
```

Comment:
This needs to be installed only when docs are built.

Reply:
Yup, done